grails-app/views/requisition/_showRequisitionItem.gsp

```diff
@@ -237,12 +237,12 @@
     </g:if>
     <g:if test="${requestTab}">
         <td class="middle center">
-            <g:set var="pickReasonCode" value="${requisitionItem?.picklistItems?.find { it.reasonCode } ?:
-                    requisitionItem?.modificationItem?.picklistItems?.find { it.reasonCode } ?:
-                    requisitionItem?.substitutionItem?.picklistItems?.find { it.reasonCode } }"/>
+            <g:set var="pickReasonCode" value="${requisitionItem?.pickReasonCode ?:
+                    requisitionItem?.modificationItem?.pickReasonCode ?:
```

Comment:
The pick reason code should only exist on the parent requisition item. In other words, even if we create a modification or substitution, the reason code should be set on the parent rather than the children.